src/governance.cpp

```diff
@@ -654,37 +659,47 @@ void CGovernanceManager::Sync(CNode* pfrom, uint256 nProp)
 
     {
         LOCK2(cs_main, cs);
-        fRateChecksEnabled = false;
-        for(object_m_it it = mapObjects.begin(); it != mapObjects.end(); ++it) {
-            uint256 h = it->first;
 
-            CGovernanceObject& govobj = it->second;
-
-            if((nProp != uint256()) && (h != nProp)) {
-                continue;
-            }
+        if(nProp == uint256()) {
```

Comment:
I think this needs a protocol bump and a peer protocol check.

Otherwise old peers won't get votes.